share/ert/forward-models/res/script/rms

```diff
@@ -52,6 +52,11 @@ def _build_argument_parser():
         default=None,
         help='the prefix of all relative paths when rms is exporting',
     )
+    parser.add_argument(
```

Comment:
Looking good :+1: Will you add the option to the RMS job description file as well?

Reply:
Something along the lines of:

```
ARGLIST <IENS>  .... <RMS_OPTS>
```

for which you can then pass `<RMS_OPTS>=-a` to have it skip env validation.